chainntnfs/interface.go

```diff
@@ -196,14 +206,28 @@ type SpendDetail struct {
 type SpendEvent struct {
 	// Spend is a receive only channel which will be sent upon once the
 	// target outpoint has been spent.
-	Spend <-chan *SpendDetail // MUST be buffered.
+	Spend chan *SpendDetail // MUST be buffered.
```

Comment:
This had to be modified when moving the logic to the `TxConfNotifier` in order to dispatch the spending details. Previously, we were declaring the spend channel outside of the `SpendEvent` creation and passing it in. We could do that here, but then we'd need to track the channels, rather than having them be a part of the notification struct. The extra code to accomplish doesn't seem worth it to me.